Userland/Services/WebServer/Client.cpp

```diff
@@ -137,6 +137,9 @@ void Client::send_response(InputStream& response, const HTTP::HttpRequest& reque
     StringBuilder builder;
     builder.append("HTTP/1.0 200 OK\r\n");
     builder.append("Server: WebServer (SerenityOS)\r\n");
+    builder.append("X-Frame-Options: SAMEORIGIN\r\n");
+    builder.append("X-Xss-Protection: 1; mode=block\r\n");
```

Comment:
I think the canonical name of this header is `X-XSS-Protection` but it probably doesn't matter.

Reply:
Correct. We probably shouldn't serve this header anyway. It sucks. And `Browser` shouldn't support it either.